pkg/gatewayserver/gatewayserver_test.go

```diff
@@ -2024,7 +2090,7 @@ func TestUpdateVersionInfo(t *testing.T) { //nolint:paralleltest
 
 	// Delete and wait for fetch interval.
 	is.GatewayRegistry().Delete(ctx, gtwIDs)
-	time.Sleep(gatewayFetchInterval << 7)
+	time.Sleep(2 * gsConfig.FetchGatewayInterval)
```

Comment:
The large interval which was required here before was caused by the said implicit jitter above. Now that is fixed, we can lower the interval here.